Classes/Command/ProjectionCommandController.php

```diff
@@ -49,54 +54,90 @@ public function listCommand()
                 $this->outputLine('PACKAGE "%s":', array(strtoupper($packageKey)));
                 $this->outputLine(str_repeat('-', $this->output->getMaximumLineLength()));
             }
-            $this->outputLine('%-2s%-40s %s', array('', $projection->getShortIdentifier(), $this->truncate($projection->getProjectorClassName())));
+            $this->outputLine('%-2s%-40s %s', array('', $projection->getShortIdentifier(), $this->shortenText($projection->getProjectorClassName())));
         }
         $this->outputLine();
     }
 
     /**
-     * @param string $projection
+     * Describe a projection
+     *
+     * This command displays detailed information about a specific projection, including the projector class name
+     * and the event types which are processed by this projector.
+     *
+     * @param string $projection The projection identifier; see projection:list for possible options
      * @return void
+     * @see neos.cqrs:projection:list
      */
     public function describeCommand($projection)
     {
-        $projection = $this->projectionManager->getProjection($projection);
-        $this->outputLine('<b>PROJECTION:</b>');
-        $this->outputLine('  <i>%s</i>', [$projection->getFullIdentifier()]);
-        $this->outputLine();
-        $this->outputLine('<b>REPLAY:</b>');
-        $this->outputLine('  %s projetion:replay %s', [$this->getFlowInvocationString(), $projection->getShortIdentifier()]);
-        $this->outputLine();
-        $this->outputLine('<b>PROJECTOR:</b>');
-        $this->outputLine('  %s', [$projection->getProjectorClassName()]);
-        $this->outputLine();
+        try {
+            $projection = $this->projectionManager->getProjection($projection);
+            $this->outputLine('<b>PROJECTION:</b>');
+            $this->outputLine('  <i>%s</i>', [$projection->getFullIdentifier()]);
+            $this->outputLine();
+            $this->outputLine('<b>REPLAY:</b>');
+            $this->outputLine('  %s projection:replay %s', [$this->getFlowInvocationString(), $projection->getShortIdentifier()]);
+            $this->outputLine();
+            $this->outputLine('<b>PROJECTOR:</b>');
+            $this->outputLine('  %s', [$projection->getProjectorClassName()]);
+            $this->outputLine();
 
-        $this->outputLine('<b>HANDLED EVENT TYPES:</b>');
-        foreach ($projection->getEventTypes() as $eventType) {
-            $this->outputLine('  * %s', [$eventType]);
+            $this->outputLine('<b>HANDLED EVENT TYPES:</b>');
+            foreach ($projection->getEventTypes() as $eventType) {
+                $this->outputLine('  * %s', [$eventType]);
+            }
+        } catch (\Exception $e) {
```

Comment:
Also we should probably just catch the relevant exceptions. otherwise we keep the server from logging potential important exceptions in the sub system